package.json

```diff
@@ -57,7 +56,7 @@
     "@typescript-eslint/parser": "^5.18.0",
     "eslint": "^8.12.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-config-next": "^12.1.4",
+    "eslint-config-next": "^12.1.1",
```

Comment:
was this downgrade for any particular reason?

Reply:
No idea why, something from the rebase most likely. 

git was a bit confused by the incoming changes. 

I'll do a manual comparison to make sure the packages match